src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js

```diff
@@ -1144,12 +1163,6 @@ define([
                         id: this.localizations[i].id,
                         title: this.localizations[i].title
                     };
-
-                    if (concreteLanguages.indexOf(this.localizations[i].id) < 0) {
-                        dropdownLocalizations[i].title += [
-                            ' (', this.sandbox.translate('content.contents.new'), ')'
-                        ].join('');
-                    }
```

Comment:
Can we really delete these lines? Then this indicator is missing, and you don't know if the page is already translated.